.circleci/config.yml

```diff
@@ -35,6 +35,15 @@ jobs:
               echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
               npm publish
             fi
+      - deploy:
+          name: Maybe Deploy GreyVest
+          command: |
+            if [ "${CIRCLE_BRANCH}" == "master" ]; then
+              echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> ~/.npmrc
+              npm run update-pkg main dist/greyVest/index.js
```

Comment:
It would probably be simpler to move this logic into the update package script. I'd rather have logic in js than in circle config